app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+end
```

Comment:
You have not implemented a logout functionality for a user to log out and log in as a different user. Kindly implement this functionality and submit it for a re-review. 
You can also refer to [this video](https://www.loom.com/share/09be395ccc8744d986d288554cebd7f5) for more information.